app/adapters/external/cache/infinite_buying_adapter.py

```
@@ -399,43 +302,14 @@ async def get_trade_statistics(self, market: str) -> TradeStatistics:
-                    success_cycles=stats_dict["success_cycles"],
-                    total_profit=Decimal(str(stats_dict["total_profit"])),
-                    total_profit_rate=Decimal(str(stats_dict["total_profit_rate"])),
-                    average_profit_rate=Decimal(str(stats_dict["average_profit_rate"])),
-                    best_profit_rate=Decimal(str(stats_dict["best_profit_rate"])),
-                    worst_profit_rate=Decimal(str(stats_dict["worst_profit_rate"])),
-                    last_updated=last_updated,
-                )
+                stats = TradeStatistics.from_cache_json(data)
                 logger.info(f"거래 통계 조회 완료: {market}")
                 return stats
         except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)